lib/govuk/dummy_content_store/content.rb

```diff
@@ -15,6 +16,13 @@ def call(env)
         example = repository.find_by_base_path(env["PATH_INFO"])
         if example
           present_example(example)
+        elsif fallback_url ||= "https://www.gov.uk/api/content"
```

Comment:
Can we configure this with environment variables?

```
DUMMY_CONTENT_STORE_FALLBACK_URL=https://www.gov.uk/api/content dummy_content_store
```

If that environment variable isn't supplied, it shouldn't configure a fallback at all